app/domain/authentication/authn_k8s/kube_exec.rb

```diff
@@ -126,12 +126,12 @@ def on_error(err)
       private
 
       def add_websocket_event_handlers(ws_client, body, stdin)
-        kubectl = self
+        kube = self
```

Comment:
Why is this needed? this looks weird.
Can't you just call `on_open`,  `on_message` etc.?

Reply:
I mean why this line is needed, not the change.
Why you need to set `self` to a var, that's my question.

Reply:
no as the brackets open a new context

Reply:
well, this was here before me but i guess we can use `self` instead. is this what you'd like?

Reply:
fine